src/instance/minecraft/CommandsManager.ts

```diff
@@ -76,6 +75,20 @@ export class CommandsManager extends EventHandler<MinecraftInstance> {
       return true
     }
 
+    if (commandName === 'help' || commandName === 'command') {
+      if (arguments_.length <= 0) {
+        await minecraftInstance.send(`/gc Commands: ${this.commands.map((command) => command.name).join(', ')}`)
+        return false
+      }
+      const command = this.commands.find((c) => c.triggers.includes(arguments_[0]))
+      if (command == undefined) return false
+
+      await minecraftInstance.send(
+        `/gc ${command.name}: ${command.description}` + `(${minecraftInstance.config.commandPrefix}${command.example})`
```

Comment:
No need to use `+`. Just use one quote. Just Bitches command for the explanation of the other change.
```suggestion
        `/gc ${command.name}: ${command.description} (${minecraftInstance.config.commandPrefix}${command.triggers[0]} ${command.example})`
```